README.md

```diff
@@ -89,24 +89,38 @@ If you execute help in your command line you can see the list of available comma
 
 ```
 khermes> help
-Khermes commands:
-  set khermes             Sets your Khermes configuration
-  set kafka              Sets your Kafka configuration
-  set template           Sets your template
-  set avro               Sets your Avro configuration
-  show config            Show all set configurations
-  ls                     Lists the nodes with their current status
-  start <node-id>        Starts event generation in node with id <node-id>
-  stop <node-id>         Stops event generation in node with id <node-id>
-  clear                  Cleans the screen.
-  help                   Shows this help.
-  exit                   Exit of Khermes Cli.
+Khermes client provide the next commands to manage your Khermes cluster:
+  Usage: COMMAND [args...]
+
+  Commands:
+     start [command options] : Starts event generation in N nodes.
+       -h, --khermes    : Khermes configuration
```

Comment:
After changing the name to khermes is a bit weird to set the khermes config with an h, should we use kh for khermes config and ka for kafka??